src/screens/CreatePost/index.js

```diff
@@ -790,7 +770,7 @@ const CreatePost = () => {
             topic={listTopic.length > 0}
             listTopic={renderListTopic()}
             label="Add Communities"
-            labelStyle={styles.hastagText}
+            labelStyle={styles.hashtagText}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `hastagText` to `hashtagText` seems to be a typo fix. This is a good change for code readability and maintainability.
```diff
- labelStyle={styles.hastagText}
+ labelStyle={styles.hashtagText}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->